### DISCUSSION TITLE
New IDF with M5Stack Core2

### DISCUSSION THREAD

**Discussion by Pedritoprint** [dl9rdz/rdz_ttgo_sonde#463] [Q&A] [answered]
I just now updated my **M5Stack Core2** with newest developper SW (dev20240905b).
 Now, if my M5Stack is scanning the Frequency channels, in the Display the channel are written one upon the other without clearing the "old" channel. So i see after short time only a white point. Is there a possible config i missed?.
My "Screen Config" in the OLED/TFT is set to 4 as in the versions before.
BTW i cannot find a screens4.txt file on my M5Stack, if i will read this file, always sreens2.txt is open.
Regards HB9CMI/p in Spain, Peter
P.S. Also 
- the Sonde-Type is writen one upon the other and 
- the automatic Import of Frequencies doesn't work
I now loaded the last Version of the old IDF and all is working as before.

**Comment by dl9rdz** [comment, marked as answer]
This was a strange effect on the M5Stack display that is not showing on the other TFT displays. Something apparently changed in the TFT library.... But I have implemented some workaround for that problem.

(While I do search all font glyphs for the real "baseline" of a font, the library does it differently by just randomly assuming the baseline is at 2/3 of the font hight. So we ended up with slightly different values, which resulted in the text begin placed at the very edge of the display. Which looked fine, but for clearing the background, the display actually tries drawing the background starting at y=-2, which is off the edge of the display. Seems like this just fails on the ILI9342 driver (so the background is not drawn), while doing the right thing on other displays.

The frequency import still is on my todo list.
